src/tools/wasm2c.cc

```diff
@@ -78,6 +78,14 @@ static void ParseOptions(int argc, char** argv) {
         s_outfile = argument;
         ConvertBackslashToSlash(&s_outfile);
       });
+  parser.AddOption(
+      'n', "module-name", "MODNAME",
+      "Unique name for the module being generated. This name is prefixes to "
```

Comment:
This help text needs more newlines to wrap nicely in the `wasm2c --help` output. Should say "prefixed" (instead of "prefixes").